dags/dag_content/nextflow_tower_functions.py

```diff
@@ -2,19 +2,20 @@
 from sagetasks.nextflowtower.utils import TowerUtils
 
 
-def create_and_open_tower_workspace(platform: str, workspace_id: str) -> TowerUtils:
+def create_and_open_tower_workspace(tower_access_token : str, platform: str, workspace_id: str) -> TowerUtils:
     """Creates and opens Nextflow Tower Workspace using sagetasks TowerUtils class
 
     Args:
-        platform (str): Which Nextflow Tower instance to run the workflow on. 
+        tower_access_token (str): Which Nextflow Tower Access Token to use for workflow run
```

Comment:
Nit: Since this is the key to the secret that holds the access token that we want, this can be misconstrued as the token itself.